### DISCUSSION TITLE
pip install not working

### DISCUSSION THREAD

**Discussion by jasonmellone** [sfu-db/connector-x#349] [Q&A]
```
$ pip install connectorx
ERROR: Could not find a version that satisfies the requirement connectorx (from versions: none)
ERROR: No matching distribution found for connectorx

```

Any clue?

**Comment by wangxiaoying** [comment]
Hi @jasonmellone , which system are you using?  If you are using one of linux/windows/macos and your python version is >=3.7, you can try to download wheel file from [here](https://pypi.org/project/connectorx/#files) for our latest version and see whether manually install the wheel file using pip works.

If there is no suitable version for you, you can also try to build the wheel file locally like [this instruction](https://sfu-db.github.io/connector-x/install.html#build-from-source-code).

**Comment by jasonmellone** [comment]
Thanks will try

On Fri, Sep 9, 2022 at 16:54 Xiaoying Wang ***@***.***> wrote:

> Hi @jasonmellone <https://github.com/jasonmellone> , which system are you
> using? If you are using one of linux/windows/macos and your python version
> is >=3.7, you can try to download wheel file from here
> <https://pypi.org/project/connectorx/#files> for our latest version and
> see whether manually install the wheel file using pip works.
>
> If there is no suitable version for you, you can also try to build the
> wheel file locally like this instruction
> <https://sfu-db.github.io/connector-x/install.html#build-from-source-code>
> .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sfu-db/connector-x/discussions/349#discussioncomment-3611928>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABQEAHP7Z2LJARKS7ONLAC3V5OPY3ANCNFSM6AAAAAAQIIDUTY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Jason Mellone <https://www.linkedin.com/in/jasonmellone/>
631.219.5351
***@***.***
